TestPlaybooks/script-TestCreateWordFile.yml

```diff
@@ -0,0 +1,45 @@
+commonfields:
+  id: TestCreateWordFile
+  version: -1
+name: TestCreateWordFile
+script: |
+  from StringIO import StringIO
+  from docx import Document
+
+  res = []
+  errEntry = {
+      "Type": entryTypes["error"],
+      "ContentsFormat": formats["text"],
+      "Contents": ""
+  }
+
+  args = demisto.args()
+  try:
+      document = Document()
+      document.add_paragraph(args['text'])
+      demisto.log('1')
```

Comment:
remove this logs or make them meaningful

Reply:
Roger that.